### DISCUSSION TITLE
Skipped detection Frames

### DISCUSSION THREAD

**Discussion by toperichvania** [blakeblackshear/frigate#14860] [Ask A Question] [answered]
Hello,

I would like to understand why I have such a high skipped detection frame rate (or is it high?). I am using deepstack running on the same machine. There are 32 cameras. Resources look quite ok. attached 
<img width="1499" alt="Screenshot 2024-11-08 at 10 52 33" src="https://github.com/user-attachments/assets/aad85766-1e20-4caf-8847-2c701bddb7fe">
<img width="1507" alt="Screenshot 2024-11-08 at 10 33 00" src="https://github.com/user-attachments/assets/8dd77265-e5c0-45a2-bc60-3ea65ac8c39e">
<img width="864" alt="Screenshot 2024-11-08 at 10 56 29" src="https://github.com/user-attachments/assets/050296a4-0b72-498d-a29d-4080bd933fa2">
<img width="638" alt="Screenshot 2024-11-08 at 10 56 52" src="https://github.com/user-attachments/assets/98348fea-2e06-40e9-a811-d4159a05063d">

**Comment by NickM-27** [comment, marked as answer]
Deepstack is too slow to support that many cameras, 32ms detection time is on the high end of okay but for that many cameras it is not fast enough.

**Comment by toperichvania** [comment]
Ah I see. Is it slow due to the way it communicates (over network even though it is running on same machine)?. 

I have RTX 4070 on this machine but frigate seem not to like it: INFO    : Detection appears to have stopped. Exiting Frigate... Not sure why as there is not much info I can get further from it...

**Reply by NickM-27** [reply]
right, network latency will affect that, it will never be as fast as interacting with the hardware directly. You'd need to make a new support request if you want to debug running tensorrt model

**Comment by toperichvania** [comment]
Thank you for your help! Much appreciated.. I have made a comment on [frigate.watchdog detection appears to have stopped](https://github.com/blakeblackshear/frigate/discussions/10764#top)
#10764 but that might not be related. Should I start a new discussion?

**Reply by NickM-27** [reply]
yes
